akash1/HTML/ASSIn3/Assignment3Q2.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>home</title>
+    <link rel="stylesheet" href="MI9.html">
```

Comment:
if u are adding css files then naming convention for css files is .css why u have used .html for css styles